table/tables/tables.go

```diff
@@ -71,6 +72,14 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 		}
 
 		col := table.ToColumn(colInfo)
+		if len(colInfo.GeneratedExprString) != 0 {
+			// here we can ignore error because GeneratedExprString has been already
+			// checked when this table or column was added.
+			var err error
+			if col.GeneratedExpr, _ = parser.ParseExpression(colInfo.GeneratedExprString); err != nil {
```

Comment:
Usually, we prefer not to put the execution line into if condition.
We can define `err1` to avoid name conflict.